index.html

```diff
@@ -391,6 +391,9 @@ <h1>Multi-Post Stories Gain+Glory</h1>
         </div>
       </div>
     </section>
+    <!-- Popup sectionk -->
+    <section class="project-popup dflex" style="display: none"></section>
+    <div class="overlay"></div>
```

Comment:
- [ ] Kindly work on the popup, it works fine but the styling does not conform to the Figma's. The image is also not displaying

### Your work
![image](https://user-images.githubusercontent.com/86200356/202531784-eb00b241-9f6d-4fe7-ba0e-5e5aa0e4cf8c.png)

### Figma 
![image](https://user-images.githubusercontent.com/86200356/202532081-87dbd64c-108f-4770-b7f7-6a75728afa1a.png)

Reply:
Hello, @Olamarx Thank you for reviewing my code,
- Please elaborate on the first change of creating the entire section dynamically. Is it part of the core requirement? I was made to understand that the requirements are to UPDATE the section using the data stored in an object, not to create it entirely from scratch in JS.
@Olamarx elaborate on this further, you always almost never elaborate on the changes you request on my reviews.

Reply:
@jaredatandi I think I did explain, it may be the case that you don't understand. What I am trying to say is that it is required that the entire work section should be implemented using javascript. You should remove the one you have in the HTML and display it with Javascript.
To do this, you will set the whole data in an array and then render it dynamically. That is what I am trying to say. If you don't understand kindly let me know so that I will explain further again.

Owoeye Olaoluwa

Reply:
Also, I would have made the comment inline but it does not show in file changed bcz the change that is not a new change so I added it to the overhaul comment.

Reply:
@Olamarx I still don't understand what you mean by "entire work section" and "whole data". The requirements clearly need the "whole data" part. Maybe you could explain this in a zoom meeting.

Reply:
@Olamarx are you there?

Reply:
Sure, let us have a meeting on zoom, please.

Reply:
You can message me on Slack.

Reply:
I am trying to understand what you mean by the entire section, right now I am unable to access zoom. Maybe later, but you can explain pretty much everything here.

Reply:
Okay, the project section is what I mean by the work section. Do you understand this?

Reply:
Not yet sir

Reply:
I think the best way is to explain on zoom. Let me know when you can access zoom, please.